src/test/java/seedu/address/logic/commands/SortCommandTest.java

```diff
@@ -24,30 +24,30 @@ public class SortCommandTest {
 
     @Test
     public void execute_nameAscendingOrder_success() {
-        executeSuccessfulSortTest(PersonComparator.NAME, true, modelAscendingDateOfLastVisit);
```

Comment:
For these tests, the model you are selecting is the model to be sorted. You have changed it to where the modeltobesorted is already sorted in the order we desire. This makes the tests not so good because we can't tell if the sorting is actually happening. Hence, it is best to leave the tests as they were or to include extra tests for checking a model that is already sorted.